GameLogic/Runtime/Cards/Card.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace BoardGame.Cards {
```

Comment:
There is already `Card.CreatureCard` defined. I don't think that we need to initialise `cardFace` and `cardBack` there, but it should be possible to reuse some logic from there and define more classes 🤷‍♀️

Reply:
I moved properties from `Card.CreatureCard` to this class.